packages/web-api/src/methods.ts

```diff
@@ -412,24 +417,6 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     },
   };
 
-  public readonly channels = {
```

Comment:
Moved these to the bottom - see also https://api.slack.com/changelog/2020-01-deprecating-antecedents-to-the-conversations-api